### PR TITLE
Add docker compose for postgres

### LINKED ISSUE #496: Update sequencer read me and have a docker-compose for postgres
Since we've implemented PostgreSQL for backup and monitoring of our sequencer, it's important to update the README to include information about PostgreSQL. Additionally, we can deploy Postgres using a docker-compose.yml file as suggested by @rakanalh.

### PR DESCRIPTION
**Pull request by rakanalh**
# Description

Makes it easy to run postgres as storage for the node.

Fixes #496

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,6 +14,7 @@ clean: ## Cleans compiled
 	@cargo clean
 
 clean-node: ## Cleans local dbs needed for sequencer and nodes
+	sudo rm -rf data/postgres
```
Comment (ercecan) [inline Makefile:17]: We should also ignore data folder in hive/dockerignore

Comment (rakanalh) [reply Makefile:17]: `.dockerignore` file added.

File: docs/run.md [hunk 2]
```diff
@@ -11,6 +11,16 @@ Build citrea:
 make build
 ```
 
+### Prequisites
+
+In order for the sequencer node to run, you need to have Postgres up and running.
```
Comment (eyusufatik) [inline docs/run.md:16]: this is not needed - if no db_config is given in the sequencer_config.toml, then the sequencer doesnt need postgres

**Review by eyusufatik — CHANGES_REQUESTED** [review]

File: docs/run.md (hunk 2 above)
Comment (eyusufatik) [reply docs/run.md:16]: I think we can just skip db stuff in readmes, only we are gonna use that feature of the sequencer

File: docs/run.md [hunk 3]
```diff
@@ -11,6 +11,18 @@ Build citrea:
 make build
 ```
 
+### Prequisites
+
+In order for the sequencer node to run with the storage DB is needed.
```
Comment (eyusufatik) [inline docs/run.md:16]: For production use cases, we leverage PostgreSQL for a few extra features in the sequencer. These features are optional, if you don't want to run them, make sure the `sequencer_config.toml` file does not contain the `db_config` altogether in order to skip using a storage DB backend such as postgres.

**Review by eyusufatik — CHANGES_REQUESTED** [review]

**Review by eyusufatik — APPROVED** [review]

**Pull request merged**